app/assets/stylesheets/content/_forms.sass

```diff
@@ -535,6 +535,11 @@ fieldset.form--fieldset
   & > [data-tooltip], & > [class^="tooltip--"]
     padding-top: 0.6rem
 
+.form--field
+  & .-break-words > .form--label
+    text-overflow: ellipsis
+    overflow: hidden
```

Comment:
I think the better approach is to introduce a `grid-invisible-overflow` mixin analogue to the `grid-visible-overflow` and replace the usage of the later for `form--label` with it. Because you are currently simply cancelling out the effect of `grid-visible-overflow` that gets applied to `form--label`. And it is probably the better approach to always apply ellipsis to form labels.